detections/aws_cross_account_activity_from_previously_unseen_account___dm.yml

```diff
@@ -0,0 +1,42 @@
+name: AWS Cross Account Activity From Previously Unseen Account - DM
+id: 21193641-cb96-4a2c-a707-d9b9a7f7792b
+version: 1
+date: '2020-05-28'
+description: This search looks for AssumeRole events where an IAM role in a different
+  account is requested for the first time.
+how_to_implement: You must install the AWS App for Splunk (version 5.1.0 or later)
+  and Splunk Add-on for AWS (version 4.4.0 or later), then configure your CloudTrail
+  inputs. Run the `Previously Seen AWS Cross Account Activity` support search only
+  once to create the baseline of previously seen cross account activity. Thanks to
+  Pablo Vega at Recurly for suggesting improvements to the search.
+type: ESCU
+references: []
+author: Rico Valdez, Splunk
+search: | tstats earliest(_time) as firstTime latest(_time) as lastTime from datamodel=Authentication_test
```

Comment:
incorrect Authentication_test, should be Authentication

Reply:
was using _test for testing. Fixed Thanks.